build.sbt

```diff
@@ -1,67 +1,162 @@
       libraryDependencies ++= Seq(
+        openapi.cross(
+          CrossVersion.for3Use2_13
+        ),
+        (algebraTestkit % Test).cross(
+          CrossVersion.for3Use2_13
+        ),
+        (algebraCirceTestkit % Test).cross(
+          CrossVersion.for3Use2_13
+        ),
+        (jsonSchemaGeneric % Test).cross(
+          CrossVersion.for3Use2_13
+        ),
         ("com.typesafe.akka" %% "akka-http" % akkaHttpVersion).cross(CrossVersion.for3Use2_13),
-        ("com.typesafe.akka" %% "akka-stream" % akkaActorVersion % Provided).cross(CrossVersion.for3Use2_13),
-        ("com.typesafe.akka" %% "akka-http-testkit" % akkaHttpVersion % Test).cross(CrossVersion.for3Use2_13),
-        ("com.typesafe.akka" %% "akka-stream-testkit" % akkaActorVersion % Test).cross(CrossVersion.for3Use2_13),
-        ("com.typesafe.akka" %% "akka-testkit" % akkaActorVersion % Test).cross(CrossVersion.for3Use2_13),
-        scalaTestDependency
+        ("com.typesafe.akka" %% "akka-stream" % akkaActorVersion % Provided).cross(
+          CrossVersion.for3Use2_13
+        ),
+        ("com.typesafe.akka" %% "akka-http-testkit" % akkaHttpVersion % Test).cross(
+          CrossVersion.for3Use2_13
+        ),
+        ("com.typesafe.akka" %% "akka-stream-testkit" % akkaActorVersion % Test).cross(
+          CrossVersion.for3Use2_13
+        ),
+        ("com.typesafe.akka" %% "akka-testkit" % akkaActorVersion % Test).cross(
+          CrossVersion.for3Use2_13
+        )
       ),
-      excludeDependencies ++= {
-        if (scalaBinaryVersion.value.startsWith("3")) {
-          List(
-            ExclusionRule("org.scala-lang.modules", "scala-collection-compat_2.13")
-          )
-        } else Nil
-      },
       versionPolicyIgnored ++= Seq(
         // Was removed from akka-http https://github.com/akka/akka-http/pull/3849
         "com.twitter" % "hpack"
       )
     )
-    .dependsOn(`algebra-jvm`, `openapi-jvm`)
-    .dependsOn(`algebra-testkit-jvm` % Test)
-    .dependsOn(`algebra-circe-testkit-jvm` % Test)
-    .dependsOn(`json-schema-generic-jvm` % "test->test")
+
+val documentation =
+  project.in(file("documentation"))
+    .enablePlugins(ParadoxMaterialThemePlugin, ParadoxPlugin, ParadoxSitePlugin, ScalaUnidocPlugin, SitePreviewPlugin)
+    .settings(
+      publish / skip := true,
+      coverageEnabled := false,
+      autoAPIMappings := true,
+      Compile / paradoxMaterialTheme := {
+        val theme = (Compile / paradoxMaterialTheme).value
+        val repository =
+          (ThisBuild / sonatypeProjectHosting).value.get.scmInfo.browseUrl.toURI
+        theme
+          .withRepository(repository)
+          .withSocial(repository)
+          .withCustomStylesheet("snippets.css")
+      },
+      paradoxProperties ++= Map(
+        "version" -> version.value,
+        "scaladoc.base_url" -> s".../${(packageDoc / siteSubdirName).value}",
+        "github.base_url" -> s"${homepage.value.get}/blob/v${version.value}"
```

Comment:
Don’t we need to also pass the `akka-version` variable?

Reply:
I checked - seems like it is needed.